README.md

```diff
@@ -106,6 +106,8 @@ Override some of the conventions:
 
 ```ruby
 Parity.configure do |config|
```

Comment:
I think this used to work in an initializer if you included parity in your Gemfile and built binstubs or your app, but doesn't apply if you are just using the generic Ruby commands or the Homebrew-installed version...